CHANGELOG.md

```diff
@@ -4,6 +4,298 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.17.0](https://github.com/go-gitea/gitea/releases/tag/v1.17.0-rc0) - 2022-06-18
+
+* BREAKING
+  * Use "main" as default branch name (#19354)
+  * use go1.18 to build gitea (#19099)
+  * Add the possibility to allow the user to have a favicon which differs from the main logo (#18542)
+  * Change initial TrustModel to committer (#18335)
+  * Upgrade Alpine from 3.13 to 3.15 (#18050)
+* SECURITY
+  * Remove deprecated SSH ciphers from default (#18697)
+* FEATURES
+  * Automatically render wiki TOC (#19873)
+  * Adding button to link accounts from user settings (#19792)
+  * feat: allow set default merge style while creating repo (#19751)
+  * Add API to serve blob or LFS file content (#19689)
+  * Auto merge pull requests when all checks succeeded via WebUI (#19648)
+  * Improve reviewing PR UX (#19612)
+  * Add API to check if team has repo access (#19540)
+  * Add support for rendering console output with colors (#19497)
+  * Add Webfinger endpoint (#19462)
+  * Add Helm Chart registry (#19406)
+  * Add Goroutine stack inspector to admin/monitor (#19207)
+  * RSS/Atom support for Repos (#19055)
+  * Add button for issue deletion (#19032)
+  * Allow to mark files in a PR as viewed (#19007)
+  * [API] Allow removing issues (#18879)
+  * add Index to comment for migrations and mirroring (#18806)
+  * Add health check endpoint (#18465)
+  * store the foreign ID of issues during migration  (#18446)
+  * Support custom ACME provider (#18340)
+  * Add packagist webhook (#18224)
+  * Add "Allow edits from maintainer" feature (#18002)
+  * Add apply-patch, basic revert and cherry-pick functionality (#17902)
+  * RSS/Atom support for Orgs (#17714)
+  * Add Package Registry (#16510)
+  * Add LDAP group sync to Teams, fixes #1395 (#16299)
+  * Pause queues (#15928)
+  * Added auto-save whitespace behavior if it changed manually (#15566)
+  * Feature: Find files in repo (#15028)
+  * Provide configuration to allow camo-media proxying (#12802)
+  * Auto merge pull requests when all checks succeeded via API (#9307)
+* API
+  * Implement http signatures support for the API (#17565)
```

Comment:
```suggestion
  * Allow removing issues (#18879)
  * Implement http signatures support for the API (#17565)
```

Reply:
Some features should be moved to enhancements.